src/task/methods/ambros_m_judges_third_place/script.py

```diff
@@ -0,0 +1,527 @@
+# This script is based on an IPython notebook:
+# https://github.com/Ambros-M/Single-Cell-Perturbations-2023/blob/main/notebooks/scp-26-py-boost-recommender-system-and-et.ipynb
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from colorama import Fore, Back, Style
+from numpy.polynomial import Polynomial
+from scipy.stats import norm, skew, kurtosis
+import scipy
+from sklearn.linear_model import Ridge
+from sklearn.decomposition import PCA
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.neighbors import KNeighborsRegressor
+from sklearn.ensemble import ExtraTreesRegressor
+from itertools import combinations
+import cupy as cp
+from py_boost import GradientBoosting
```

Comment:
Is there an alternative to GradientBoosting we can use when there is no GPU available?